napari_plugin_manager/qt_package_installer.py

```diff
@@ -241,12 +241,16 @@ class InstallerQueue(QProcess):
 
     # emitted when all jobs are finished
     # not to be confused with finished, which is emitted when each job is finished
-    allFinished = Signal()
+    allFinished = Signal(object)
+    processFinished = Signal(
+        int, int, object, object
+    )  # exit_code, exit_status, action, pkgs
```

Comment:
```suggestion
    # exit_code, exit_status, action, pkgs
    processFinished = Signal(int, int, object, object)  
```

I dislike this formatting artifact 😬

Reply:
Fixed